analytics/run-rbqm-analytics.R

```diff
@@ -0,0 +1,75 @@
+# TODO:
+# - tweak data to introduce greater outliers
+# - add explanatory comments to script and to workflows
+
+library(purrr)
+library(dplyr)
+library(gsm)
```

Comment:
We should write this toward the split version of gsm, right (so the training isn't out of date before we give it)? I updated gsm.app@dev to use gsm + gsm.kri in anticipation of the split. I definitely want to make sure we're all using the same versions so they don't have to change installations mid-workshop.

Reply:
done, using the `dev` branches of everything except `gsm`, which is on the `gsm-core-test` branch